ui-tests/src/test/java/org/openmrs/reference/DeleteAppointmentBlockTest.java

```diff
@@ -1,53 +1,60 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
 
 
 package org.openmrs.reference;
 
 import org.junit.After;
 import org.junit.Before;
-import org.junit.Ignore;
 import org.junit.Test;
-import org.openmrs.reference.page.*;
-import org.openmrs.uitestframework.test.TestBase;
-import static org.junit.Assert.assertNotNull;
-
-
-/**
- * Created by nata on 08.07.2015.
- */
-public class DeleteAppointmentBlockTest extends TestBase {
-    private HomePage homePage;
-    private HeaderPage headerPage;
-    private AppointmentBlocksPage appointmentBlocksPage;
-
+import static org.junit.Assert.assertTrue;
+
+public class DeleteAppointmentBlockTest extends ManageProviderSchedulesTest {
+    
+	private String correctStartTimeFirtAppointment = "08";
```

Comment:
Instead of formatting whole files, can you just do just the same tab indentions for the obvious few lines where you had your changes like the places i mentioned?